template/assets/js/modules/reservation.js

```diff
@@ -208,147 +216,6 @@ function addReservation(){
       });
   });
 
-  /*add_form = document.createElement( 'form' ), // form element which is added
-  reservations = new Reservation, // class instance
```

Comment:
Waarom is hier zoveel weg?

Reply:
HTML van deze module wordt nu geladen op regel 36 en is aanmaken elementen met JS is nu overbodig